docs/login_and_acl.md

```diff
@@ -235,6 +251,7 @@ The following environment variables are used to setup the optional login and doc
 - `AZURE_ENFORCE_ACCESS_CONTROL`: Enforces Entra ID based login and document level access control on documents with access control assigned. Set to true before running `azd up`. If `AZURE_ENFORCE_ACCESS_CONTROL` is enabled and `AZURE_ENABLE_UNAUTHENTICATED_ACCESS` is not enabled, then authentication is required to use the app.
 - `AZURE_ENABLE_GLOBAL_DOCUMENT_ACCESS`: Allows users to search on documents that have no access controls assigned
 - `AZURE_ENABLE_UNAUTHENTICATED_ACCESS`: Allows unauthenticated users to access the chat app, even when `AZURE_ENFORCE_ACCESS_CONTROL` is enabled. `AZURE_ENABLE_GLOBAL_DOCUMENT_ACCESS` should be set to true to allow unauthenticated users to search on documents that have no access control assigned. Unauthenticated users cannot search on documents with access control assigned.
+- `AZURE_DISABLE_APP_SERVICES_AUTHENTICATION`: Disables [use of the integrated authentication for App Services](https://learn.microsoft.com/azure/app-service/overview-authentication-authorization). Built-in authentication to the chat app is used instead. Useful when you want to provide programmatic access to the chat endpoints with authentication.
```

Comment:
```suggestion
- `AZURE_DISABLE_APP_SERVICES_AUTHENTICATION`: Disables [use of built-in authentication for App Services](https://learn.microsoft.com/azure/app-service/overview-authentication-authorization). An authentication flow based on the MSAL SDKs is used instead. Useful when you want to provide programmatic access to the chat endpoints with authentication.
```

Reply:
The docs usually call it "built-in" so am avoiding calling our MSAL flow "built in".